### DISCUSSION TITLE
Model With Boundary Condition Issues

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#1914] [Q&A]
Hello All,

I am having weird boundary condition issues in my Radioss simliation and seeking for help here.  Basically I am converting one Abaqus copper rod impact example to radioss. I am assumed that I applied the exact B.Cs in Radioss compared to Abaqus however the rod did not move at all in my radioss animation. Could you please holp me identify where is the issue in my Radioss model? I have upload both the Abaqus input deck and Radioss files. Thanks in advance again. 
[rodimpac3d_f.zip](https://github.com/OpenRadioss/OpenRadioss/files/12751689/rodimpac3d_f.zip)

Regrads
Lee

**Comment by MarianBulla** [comment]
Hi Lee,
thanks for your interestin OpenRadioss.
Regarding your model:
1. You have a duplicate node groupe No. 11.
I have deleted the empty one.
2. You fixed all translational degree of freedom in your 3rd /BCS for the ground nodes. So your structure can't move in the direction of your initial velocity (see image).
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/0c0b5362-c5df-4543-b86c-ec4f8a4c5c97)

Hope it helps you?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Thanks for your help. 1) That was a mistake, Thanks 2). please see attached in BCs #3, it should just fix the z direction movement right? why you said I fixed all translation DOFs? In ABAAQUS example, the z direction should be fixed and works well (please see the deformation of abaqus results). 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/c93874b6-a02e-4738-8264-ef7b9b5ed0b6)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/e5cc3698-a550-473c-a1f9-faca655ccebc)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/e04fe37b-d6e7-4a17-8218-d28dc71f782b)

**Reply by li2qn** [reply]
Please help on this. Thanks

**Reply by li2qn** [reply]
For more information, see the attached image. Even if I release all the all the DOFs on BCS #3, only a few nodes have the define the initial velocity (227 m/s). 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/9add2713-97e9-418e-a6cf-308f24a186d8)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/0463ded2-4ce0-43fb-ae98-01a9de8e42d1)

**Comment by li2qn** [comment]
Hello, any chance you guys who is an expert of Radioss can help me on this? Thanks so much for any suggestions/comments.

**Comment by MarianBulla** [comment]
Hello Lee,
I think you had an issue with the definition of the initial velocity. Not all nodes of your part were in that group.
I have modified your initial velocity definition:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/cc45f98b-f285-4421-9315-d6b14b16050d)

Same for your material hardening furnction, which was defined only with 1 point. 
It makes it difficult for extrapolation, because Radioss needs at least a slope, where it extrapolates curves. 
So I changed your material stress-vs.-plastic-strain curve to this (add a 2nd point):
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/3af5a168-854c-4c4a-9c8a-9c71b7a49cbb)


The result is:

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/d49ef6a3-3aa4-4a41-a971-8e4e117bf606)

The plastification is already huge. Maybe the amount of energy is too large ?
The mesh is already at its limit.
A better hardening curve should solve it. Or you can swith to SPH or ALE ?

Hope it helps you ?

Thanks and best regards,
Marian

[mod_2.zip](https://github.com/OpenRadioss/OpenRadioss/files/12756130/mod_2.zip)

**Reply by li2qn** [reply]
Thanks so much for your investigation. That helps me furture understand Radioss more. Questions about the comparation between ABAQUS and Radioss, I am wandering what is the theory's reason in behand of Radioss causing it's not able to handle this type of application well as in ABAQUS? If you see the attached ABAQUS input deck, it works very well with exactly same mesh and material without the needed to extend the hardening curve? Thanks for your comments again

**Comment by RevolT87** [comment]
Hi @li2qn,
I agree with Marian, you should definitively use an ALE grid formulation for such application. Please try to set Iale=1 in the /PROP/TYPE14. an automatic grid control will be activated. 
Regards,
Mathis.

**Reply by li2qn** [reply]
Many Thanks for your comments. It's very very helpful

**Reply by li2qn** [reply]
BTW, I do not see Iale option in /PROP/TYPE14, is this available in any other cards? Thanks

**Reply by RevolT87** [reply]
Hi @li2qn,
Iale is a very new flag in the /PROP/SOLID. Therefore it is avalaible in OpenRadioss and not yet in Altair Radioss. Hence the documentation of the flag is not yet released. 
It is the 3rd field of the 3rd line of the /PROP/SOLID keyword. 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/104916733/1128abef-f327-4187-b35d-8d231f11bbf4)

I hope it helps.
Regards,
Mathis

**Comment by Schwoertzig** [comment]
Hi @li2qn ,
you will also have to define boundary conditions for grid velocity with /ALE/BCS
![rod_ALE](https://github.com/OpenRadioss/OpenRadioss/assets/104987175/10f7dd6b-d450-4b2a-96a5-395867423b5b)
[Taylor_rod_ALE.tar.gz](https://github.com/OpenRadioss/OpenRadioss/files/12762231/Taylor_rod_ALE.tar.gz)
Please find attached an input file using ALE framework instead of lagrangian one.
Thierry

**Reply by li2qn** [reply]
Many Thanks. That's very very helpful. I am wondering why the same model works very well in ABAQUS even in  Lagrangian domain? Any particular reason in theory behand Radioss that makes it harder to simulate?

**Reply by MarianBulla** [reply]
Hi Lee,
it can have several reasons. OpenRadioss is NOT Abaqus, even if the roots are the same. There are differneces, which were developed and implemented separately in all the solvers. It can be on the material definition and/or element formulations, which are optimized for different applications. You can review the source code of OpenRadioss and get deeper insights on whats happening there.

Regarding the usability of the ALE approach, please review closer the model provided by @Schwoertzig 

Hope it helps you ?

Thanks and best regards,
Marian
